health/health.d/nut.conf

```diff
@@ -26,8 +26,8 @@ component: UPS
    lookup: average -60s unaligned of battery_charge
     units: %
     every: 60s
-     warn: $this < 100
-     crit: $this < (($status == $CRITICAL) ? (60) : (50))
+     warn: $this < 75
+     crit: $this < 25
```

Comment:
I'm spoiled. I never saw server installation that was not acting on power loss automatically. Usually they know that they need to shut off. I agree tho that it might be bumped, I probably went too low.

Reply:
as for the hysteresis - I think you not only do not need it but it is beneficial to not have it. If power is flapping all the time and you are constantly charging and discharging - it can also alert you. In normal circumstances you either charge or discharge.

Reply:
Fair point. Let's just increase then the critical value (40%?) and I will mark this one as resolved.

Reply:
You bargain good :D will do so